docs/array.md

```diff
@@ -0,0 +1,88 @@
+# array
+
+* `copyWithin` - Copies a sequence of elements to another position in the given array
```

Comment:
Subheadings would be easier to read than bullets. Bullets don't really provide good visual separation, especially when they're buried in code examples.

In any case, be consistent. `array` and `string`, for example, use bulleted lists of method names. `Map`, on the other hand, uses a cookbook style. Either, or even a combination of both, would be fine, but all the module docs should follow the same format.